### ISSUE TITLE
Could you please help to convert the pth models to onnx models?

### ISSUE THREAD

**Issue by mehao** [cszn/USRNet#43]
Could you please help to convert the pth models to onnx models?
Thanks

**Comment by Jerry-Master** [comment]
Yes, please. I also have trouble converting. I have added an issue to torch in case it is something to fix: [128324](https://github.com/pytorch/pytorch/issues/128324).
